src/characteristicMeanings.js

```diff
@@ -0,0 +1,45 @@
+
```

Comment:
The modularization used here by creating a separate file is really great. I will think about doing this with any large data I have.